applications/sdp/mspi/src/main.c

```diff
@@ -12,26 +12,31 @@
 
 #include <hal/nrf_vpr_csr.h>
 #include <hal/nrf_vpr_csr_vio.h>
+#include <hal/nrf_vpr_csr_vtim.h>
 #include <haly/nrfy_gpio.h>
 
-#include <zephyr/drivers/mspi.h>
+#include <drivers/mspi/nrfe_mspi.h>
 
-#define MAX_DATA_LEN 256
+#include <stdint.h>
 
-#define XFER_COMMAND_IDX (0)
-#define XFER_ADDRESS_IDX (1)
-#define XFER_DATA_IDX	 (2)
+/*************************************************
+ * 		   DEFINES
+ *************************************************/
```

Comment:
I think these types of comments can be removed, we usually stick to one order of things in source files (defines, macros, typedefs, etc.), so I don't think this is necessary.

Reply:
they make it easier to find functions scrolling as they are quite big and catch attention. if it is against some guidelines I will remove them, but if not I would prefer to leave them as it improves readability

Reply:
removed, reorganized code